Component/Pet/Pet.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+const StyledPet = styled.div`
+  display: flex;
+  border: 2px solid #000000;
+  border-radius: 10px;
+  margin: 10px;
+  padding: 10px;
+  justify-content: space-around;
+  align-items: center;
+  max-width: 250;
+  background-color: var(--secondary-color);
+  span {
+    font-size: 20vw;
+    position: absolute;
+  }
+  .portrait {
+    width: 45vw;
+    height: 30vh;
+    border-radius: 100%;
+    background: hotpink;
```

Comment:
In a demo it is fine to use color names. In all other projects I strongly suggest to use color codes (hex, rgb, hsl).